core/src/main/scala/cats/data/Xor.scala

```diff
@@ -166,9 +166,19 @@ private[data] sealed abstract class XorInstances extends XorInstances1 {
       def combine(x: A Xor B, y: A Xor B): A Xor B = x combine y
     }
 
-  implicit def xorBifunctor: Bifunctor[Xor] =
-    new Bifunctor[Xor] {
+  implicit def xorBifunctor: Bifunctor[Xor] with Bifoldable[Xor] =
```

Comment:
The `Bifunctor[Xor]` instance was there prior to this PR. It wasn't in other places so I figured I would do those separately, hence https://github.com/typelevel/cats/pull/867. However because it looks like we have a use case for https://github.com/typelevel/cats/issues/800 I may abandon #867  and add `Bitraverse` once this is OK'ed and merged :-)